<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teampage</class>
 <widget class="QDialog" name="teampage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
                     stop:0 #C7FFC7, stop:1 #B5FFB5);
background-image: url(:/images/Untitled design (3).png);
background-repeat: no-repeat;
 background-position: center;</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_companyusername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_companypassword">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Login Now</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Enter your Company/Team Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;</string>
   </property>
   <property name="text">
    <string>Enter your Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
